src/shared/components/CategoryPicker/CategoryPicker.tsx

```diff
@@ -1,4 +1,5 @@
 import React, { useState, useRef } from 'react'
+import * as _ from 'lodash'
```

Comment:
We shouldn't use `import *` syntax if not necessary. Doing that will force the full library to get included in the built bundle, increasing its size. If you import only specific members, like `import { cloneDeep } from 'lodash`, Webpack should be able to do treeshaking and only include the required members in the bundle.